triton/build_engine.sh

```diff
@@ -0,0 +1,24 @@
+
+#!/bin/bash
+
```

Comment:
The root cause of SA's issue should be related to wrong default runtime of docker when [building the image](https://stackoverflow.com/questions/59691207/docker-build-with-nvidia-runtime). Will it be better to keep one-click script and teach customer how to access GPU in the building time ?

Reply:
Previously I used docker version is `24.0.6` and set the default docker runtime to nvidia docker but it didn't help. Now with `DOCKER_BUILDKIT=0 ` in the blog your shared, the docker build works smoothly.

docker `20.10` doesn't have this issues with the nvidia docker as default runtime.

However, there is a warning that the `DOCKER_BUILDKIT=0 ` will be DEPRECATED, so we may face the same error in future with latest docker version, I suggestion we keep this build_engine.sh for users who are using latest version of docker and haven't set the nvidia docker as default runtime.

how do you think?

Reply:
OK, I didn't know the deprecation warning. That makes sense.